python/mvdtool.cpp

```diff
@@ -473,6 +473,52 @@ PYBIND11_MODULE(mvdtool, mvd) {
                 return _atIndices<std::string>(func, f.size(), idx);
              })
         .def_property_readonly("all_layers", &SonataFile::listAllLayers)
+        .def("hasAttribute", [](const SonataFile& f, const std::string& name){
+                return f.hasAttribute(name);
+             })
+        .def("getAttribute", [](const SonataFile& f, const std::string& name) {
+                const std::string dtype = f.getAttributeDataType(name);
+                if (dtype == "double" || dtype == "float") {
+                    auto res = f.getAttribute<double>(name);
+                    return py::array(res.size(), res.data());
+                } else if (dtype == "string") {
+                    auto res = f.getAttribute<std::string>(name);
+                    return py::array(py::cast(res));
```

Comment:
I thought we could return without any conversion, and on de side of Python it would be a python list of strings, which is IMO ok. You prefer always numpy?

Reply:
I also tried without conversion but the compiler complains 
```inconsistent types 'pybind11::array' and 'std::vector<std::__cxx11::basic_string<char> >' deduced for lambda return type``` 
Apparently the lambda function only accepts one return type, then numpy to adapt to all cases

Reply:
Just for a reminder, the same reason as above, because one lambda function accepts only one return type, I can not bind the function 
```getAttribute(self: mvdtool.sonata.File, name: str, offset: int) -> float/int/string ``` 
like the existing functions:
```layers(self: mvdtool.sonata.File, arg0: int) -> str```
```exc_mini_frequencies(self: mvdtool.__File, arg0: int) -> float```
Alternatively, we can call```->getAttribute("layer", 3, 0)``` to return a numpy array with only one element.